### ISSUE TITLE
Problem with validation set

### ISSUE THREAD

**Issue by friedelwolff** [Calamari-OCR/calamari#28]
I'm training models successfully. Thank you for Calamari!

I am not able to implement early stopping during training through the ``--validation option``. Training with ``calamari-cross-fold-train`` gives the same problem: It never scores above 0 on the validation set. To debug, I printed the predictions on the validation set, and they consistently look like the output of a freshly initialised network (like the output at iteration 0 during training). In ``ocr/trainer.py`` I can't see where ``test_net`` would get any weights other than the randomly initialised ones. Nothing from ``train_net`` seems to touch the prediction on the validation set. I guess I might be missing something.

Is this a known regression, or something I'm missing? Thank you for your help.

**Comment by ChWick** [comment]
There is a problem with the current pip package if you use Tensorflow >= 1.8. Either Downgrade tensorflow to 1.7 (pip uninstall tensorflow(_gpu)  && pip install tensorflow(_gpu)==1.7) or install calamari using the github code (recommended). We will soon upgrade the pip package to the current version.

**Comment by friedelwolff** [comment]
Thank you! I installed calamari trunk and it seems to be resolved.
